packages/ndla-licenses/src/licenseRights.js

```diff
@@ -180,6 +181,26 @@ const cc = {
   },
 };
 
+const na = {
+  short: NA,
+  nn: {
+    title: 'NA - ikkje relevant',
+    userFriendlyTitle: 'NA - ikkje relevant',
+    description: 'Lisensen er ment for begreper som ikkje trenger lisens.',
```

Comment:
Jeg tror ndla har bestemt seg for at det skal hete forklaringer istedet for begreper(?)
@gunnarvelle vet sikkert dette bedre.

Reply:
Jepp. Ordet begrep finnes ikkje i ed lenger. Når det er sagt kan det fort tenkes at dette ønskes brukt andre plasser, så skriv heller innhold som ikkje treng lisens.
```suggestion
    description: 'Lisensen er meint for innhald som ikkje treng lisens.',
```